### ISSUE TITLE
error message when hireclim2 computes Nino3.4 SST 

### ISSUE THREAD

**Issue by plesager** [plesager/ece3-postproc#1]
at high res (ORCA025) the nemo_post.sh log reports:

```
##[16:24:52] master_hiresclim.sh[189]> /usr/local/apps/cdftools/3.0/bin/cdfmean aGez_1m_19890101_19891231_grid_T.nc sosstsst T 470 668 519 479 0 0
 npiglo =          199
 npjglo =          -39
 npk    =           75
 npt    =           12
 nvpk   =            1
 depth dim name is deptht
  Problem in getvar for e1t
 ERROR in NETCDF routine, status=         -57
 NetCDF: Start+count exceeds dimension bound 
```                                   
Note that that does not prevent the code to write a EXP_YEAR_sosstsst_nino34.nc file.

**Comment by oloapinivad** [comment]
We saw the same issue a few ago, I think it is related to a mismatch between the points provided to compute the ENSO region and the ORCA025 grid for NEMO 3.6 that is different - as long as I understand - to the older one (that was used in older EC-Earth versions). 

Now that we have Barakuda operational I think it is not very important, but still it would be nice to fix it.

**Comment by abalanzino** [comment]
Hi Philippe,

as wrote Paolo, I obtained new Nino3.4 coordinates using Barakuda operational (I added a print in one of the python modules). So the code line ( ORCA025L75 ) in nemo_post.sh should be:
 
Nino3.4 region (in ij coordinates)
   nino34_region='469 669 507 547'

**Comment by etiennesky** [comment]
I had a problem in my installation with the Nino3.4 coordinates of the ORCA1 grid, am I the only one?

**Comment by abalanzino** [comment]
The correct Nino3.4 coordinates of the ORCA1 grid should be:
Nino3.4 region (in ij coordinates)
    nino34_region='119 168 133 161'

**Comment by etiennesky** [comment]
Great thanks! right now the coordinates are the same as for the ORCA025 grid.

nino34_region='470 668 519 479'

**Comment by abalanzino** [comment]
Yes, there was a mistake!
Moreover the coordinates (ORCA025) nino34_region='470 668 519 479' relate to old mesh.nc and mask.nc, when the grid was 1442x1021 (now the grid for nemo3.6 is 1442x1050 and the coordinates are '469 669 507 547').

**Comment by plesager** [comment]
Great! It may be worth documenting somewhere in the README how to retrieve these coordinates if someone changes the mesh grid. Actually I was wondering if we could make that automatic?

**Comment by abalanzino** [comment]
I think it is possible documenting in the README. 
We still haven't thought about how to make that automatic, but we could work on that one.

**Comment by plesager** [comment]
Thank you @abalanzino, I put your fixes in  5cf743d.
